lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -642,6 +697,10 @@ e2function entity entity:passenger()
 	return this:GetPassenger(0)
 end
 
+e2function entity entity:getLastSpawnpoint()
+	return IsValid(this) and this:GetTable().LastSpawnpoint or NULL
```

Comment:
> What is LastSpawnpoint?

Yeah. This part is a bit undocumented...
Basically, it is set on a Player object when a player is spawned in, `LastSpawnpoint` is set to the entity that's pretty much what it says "the last spawn-point entity for a given player" (i.e. it returns an "info_player_start" entity on gm_flatgrass map; there are 56 spawn-point entities on the gm_flatgrass map, a suitable spawnpoint is detected each time a player wants to (re)spawn and its code can be browsed in the "base" gamemode)...
For the reference, see the [base gamemode code here](https://github.com/Facepunch/garrysmod/blob/cca629d87bb3a6b89b188887561d63fc13bed9c0/garrysmod/gamemodes/base/gamemode/player.lua#L385-L493).
So, this should work for any gamemode which is base-derived (this includes the Sandbox) gamemode. Even if it fails to obtain it, it will return NULL entity (I will push a new commit to this e2function in a moment).